### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by 8Men5Camels**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,30 @@
...
+    mocked_rate_prediction.return_value = 25.23
+
+    assert cryptocurrency_action(24.5) == "Do nothing"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_rate_is_slightly_lower(mocked_rate_prediction):
+    mocked_rate_prediction.return_value = 23.34
+
+    assert cryptocurrency_action(23.82) == "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:30]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Good job!
